packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -216,22 +214,22 @@ const virtualService = new appmesh.VirtualService(stack, 'service-1', {
 node.addBackend(appmesh.Backend.virtualService(virtualService));
 ```
 
-The `listeners` property can be left blank and added later with the `node.addListener()` method. The `healthcheck` and `timeout` properties are optional but if specifying a listener, the `port` must be added.
+The `listeners` property can be left blank and added later with the `node.addListener()` method. The `healthcheck` and `timeout` properties are optional, but the `port` must be specified when specifying a listener.
```

Comment:
```suggestion
The `listeners` property can be left blank and added later with the `node.addListener()` method. The `serviceDiscovery ` must be specified when specifying a listener.
```

Actually `port` is also optional even when using `node.addListener()`.

I think `healthcheck` and `timeout` reference can also be removed. Even though they actually are optional properties for `listener`, all properties for `listener` are optional.

Instead, I think useful information to add here is the call out that the `serviceDiscovery` for a `VirtualNode` must be specified when `listeners` is specified. ([reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-virtualnode-virtualnodespec.html))

Reply:
To mention, this discrepancy should've been addressed earlier.

Reply:
@Jerry-AWS Looks like you reverted back the change. Was this interntional?

Reply:
@Jerry-AWS Looks like you reverted back the change. Was this intentional? #